service/evaluation/evaluation.go

```diff
@@ -661,20 +662,23 @@ func (svc *Service) evaluateControl(ctx context.Context, auditScope *orchestrato
 			status, nonCompliantAssessmentResults = handleCompliant(r)
 		case evaluation.EvaluationStatus_EVALUATION_STATUS_NOT_COMPLIANT, evaluation.EvaluationStatus_EVALUATION_STATUS_NOT_COMPLIANT_MANUALLY:
 			// Evaluation status does not change if it is already not_compliant
-			nonCompliantAssessmentResults = append(nonCompliantAssessmentResults, r.GetFailingAssessmentResultIds()...)
+			nonCompliantAssessmentResults = append(nonCompliantAssessmentResults, r.GetAssessmentResultIds()...)
```

Comment:
Why are the nonCompliantAssessmentResults no longer included in the Evaluation Result? I would prefer to have that information in the result.

Reply:
All results are now in the evaluation result

Reply:
But the idea was that we know which assessment results are responsible for negative evaluation results. 
Anyway, one can also manually extract the information again. In any case, I would rename the nonCompliantAssessmentResults to assessmentResults or something similar.